Shifty.App/Program.cs

```diff
@@ -42,10 +43,22 @@ public static void ConfigureServices(IServiceCollection services)
                 .AddHttpMessageHandler<RequestAuthenticationHandler>();
             services.AddScoped(provider => 
                 new AnalogCoreV1(provider.GetRequiredService<IHttpClientFactory>().CreateClient("AnalogCoreV1")));
+            
+            services.AddScoped(sp => sp.GetRequiredService<IHttpClientFactory>()
+                .CreateClient("AnalogCoreV2"));
+            services.AddHttpClient("AnalogCoreV2",
+                    client => client.BaseAddress = new Uri("https://core.dev.analogio.dk/"))
```

Comment:
For this to be production ready, we also need to find a way to load these from config instead. Could you for now at least assign both clients to a const string, instead?

Reply:
The loading from appconfig doesn't have to be part of this pr though, that can be done later

Reply:
This has been addressed in PR #18 
@A-Guldborg will need to rebase this branch on account of that